libs/utils/samplerum.js

```diff
@@ -28,8 +28,10 @@ export function sampleRUM(checkpoint, data = {}) {
   try {
     window.hlx = window.hlx || {};
     if (!window.hlx.rum) {
-      const usp = new URLSearchParams(window.location.search);
-      const weight = (usp.get('rum') === 'on') ? 1 : 100; // with parameter, weight is 1. Defaults to 100.
+      const weight = (window.SAMPLE_PAGEVIEWS_AT_RATE === 'high' && 10)
+      || (window.SAMPLE_PAGEVIEWS_AT_RATE === 'low' && 1000)
+      || (new URLSearchParams(window.location.search).get('rum') === 'on' && 1)
+      || 100;
       const id = Array.from({ length: 75 }, (_, i) => String.fromCharCode(48 + i)).filter((a) => /\d|[A-Z]/i.test(a)).filter(() => Math.random() * 75 > 70).join('');
```

Comment:
we have switched to a simpler and less prone to collisions, logic to generate ids.
you may want to change it here as well https://github.com/adobe/helix-rum-js/blob/80a9e5e041ad1f935926c888da9dcccce4abb250/src/index.js#L20

Reply:
Added 👍